app/styles/app.scss

```diff
@@ -87,3 +87,22 @@ body {
 .btn btn-danger, btn-primary, a, hover {
   color: $white;
 }
+
+.item{
+    background: #333;    
+    text-align: center;
+    height: 300px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces